src/substrate/types.ts

```diff
@@ -72,6 +98,7 @@ export enum EntityKind {
   TreasuryProposal = 'treasury-proposal',
   CollectiveProposal = 'collective-proposal',
   SignalingProposal = 'signaling-proposal',
+  Bounty = 'bounty',
```

Comment:
I would call it a TreasuryBounty rather than just "bounty".

Reply:
👍